idaes/surrogate/keras_surrogate.py

```diff
@@ -16,7 +16,10 @@
 import numpy as np
 import json
 import os.path
-import tensorflow.keras as keras
+
+from pyomo.common.dependencies import attempt_import
+keras, keras_available = attempt_import('tensorflow.keras')

```

Comment:
One more quick change: by default, the error message emitted by `attempt_import` is:
```
The tensorflow.keras module (an optional Pyomo dependency) failed to import: ModuleNotFoundError: No module named 'tensorflow'
```
If we don't want this to blame Pyomo, then we can specify our own message:
```
keras, keras_available = attempt_import(
    'tensorflow.keras', 'The tensorflow.keras module (an optional IDAES dependency) failed to import'
)